golos.publication/golos.publication.cpp

```diff
@@ -885,14 +885,12 @@ void publication::reblog(name rebloger, structures::mssgid message_id, std::stri
 }
 
 int64_t publication::pay_delegators(int64_t claim, name voter,
-        eosio::symbol tokensymbol, std::vector<structures::delegate_voter> delegate_list) {
+        eosio::symbol tokensymbol, std::vector<structures::delegate_voter> delegate_list, structures::mssgid message_id) {
     int64_t dlg_payout_sum = 0;
     for (auto delegate_obj : delegate_list) {
         auto dlg_payout = claim * delegate_obj.interest_rate / config::_100percent;
-        INLINE_ACTION_SENDER(golos::vesting, paydelegator) (config::vesting_name,
-            {config::vesting_name, config::active_name},
-            {voter, eosio::asset(dlg_payout, tokensymbol), delegate_obj.delegator,
-            delegate_obj.payout_strategy});
+        payto(delegate_obj.delegator, eosio::asset(dlg_payout, tokensymbol), static_cast<enum_t>(payment_t::VESTING), get_memo("delegator", message_id));
```

Comment:
what about payout strategy?

Reply:
@zxcat it was discussed with @s-medvedev , and it was decided that the implementation would be such